src/Atmos/Model/linear.jl

```diff
@@ -0,0 +1,71 @@
+abstract type AtmosLinearModel <: BalanceLaw
+end
+
+
+vars_state(lm::AtmosLinearModel, T) = vars_state(lm.atmos,T)
+vars_gradient(lm::AtmosLinearModel, T) = @vars()
+vars_diffusive(lm::AtmosLinearModel, T) = @vars()
+vars_aux(lm::AtmosLinearModel, T) = vars_aux(lm.atmos,T)
+vars_integrals(lm::AtmosLinearModel,T) = @vars()
+
+
+update_aux!(dg::DGModel, lm::AtmosLinearModel, Q::MPIStateArray, auxstate::MPIStateArray, t::Real) = nothing
+integrate_aux!(lm::AtmosLinearModel, integ::Vars, state::Vars, aux::Vars) = nothing
+flux_diffusive!(lm::AtmosLinearModel, flux::Grad, state::Vars, diffusive::Vars, aux::Vars, t::Real) = nothing
+function wavespeed(lm::AtmosLinearModel, nM, state::Vars, aux::Vars, t::Real)
+  return soundspeed_air(Float64(T_0))
+end
+
+boundary_state!(nf, lm::AtmosLinearModel, x...) = nothing
```

Comment:
Shouldn't we need boundary conditions for the linear problem? @fxgiraldo or @mwarusz

Reply:
Yes, you need linear boundary conditions. But keep in mind that the wave modes resolved here transport nothing. Hence, the boundary conditions are not the same as for the transport term. Just use linear reflecting boundary conditions.